Delete.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Do you mind changing the filenames to lowercase for consistency across our other projects?